src/main/java/com/palantir/docker/compose/execution/DockerComposeExecOption.java

```diff
@@ -16,15 +16,20 @@
 package com.palantir.docker.compose.execution;
```

Comment:
Sorry, should've mentioned this yesterday - these should probably be @Immutables classes so someone doesn't spend 3 hours trying to figure out a weird bug because these don't implement equals properly. I'm happy to merge this then have a follow up PR/do it myself.